### PR TITLE
feature/dgs-10-announcement

### PR DESCRIPTION
**Pull request by paulbakker**

### CODE REVIEW

**Review by iparadiso — APPROVED** [review]

**Review by asibross — APPROVED** [review]

**Comment by srinivasankavitha** [comment]
Doesn't have to be part of this PR, but separately this needs to be updated: https://netflix.github.io/dgs/advanced/subscriptions/

**Review by srinivasankavitha — APPROVED** [review]

**Comment by paulbakker** [comment]
> Doesn't have to be part of this PR, but separately this needs to be updated: https://netflix.github.io/dgs/advanced/subscriptions/

updated

**Review by srinivasankavitha — APPROVED** [review]

File: docs/error-handling.md [hunk 1]
```diff
@@ -12,23 +12,42 @@ For some specific exception types, a more specific GraphQL error type is used.
...
-Mapping custom exceptions
------
 
+# Handling exceptions
 It can be useful to map application specific exceptions to meaningful exceptions back to the client.
-You can do this by registering a `DataFetcherExceptionHandler`.
+The framework provides two different mechanisms to achieve this.
+
+## Mapping custom exceptions with @ControllerAdvice
+The easiest way to map manually map exceptions is to use `@ControllerAdvice` from Spring for GraphQL.
```
Comment (srinivasankavitha) [inline docs/error-handling.md:21]: map manually map => manually map

File: docs/error-handling.md [hunk 2]
```diff
@@ -12,23 +12,42 @@ For some specific exception types, a more specific GraphQL error type is used.
...
+Annotate a class with `@ControllerAdvice`, and add a method annotated with `@GraphQLExeptionHandler` for each type of exception you want to handle.
```
Comment (srinivasankavitha) [inline docs/error-handling.md:22]: typo in `@GraphQlExeptionHandler`

File: docs/error-handling.md [hunk 3]
```diff
@@ -12,23 +12,42 @@ For some specific exception types, a more specific GraphQL error type is used.
...
+The method must take an argument of the type of exeption it should handle, and must return `GraphQLError`.
```
Comment (srinivasankavitha) [inline docs/error-handling.md:23]: typo exeption => exception

**Pull request merged**